idd/Energy+.idd.in

```diff
@@ -50246,6 +50246,161 @@ ZoneHVAC:EquipmentConnections,
+       \min-fields 10
+   A1, \field Name
+       \required-field
+       \reference SpaceSplitterNames
+   A2, \field Zone Name
+       \note Must be a controlled zone which has a ZoneHVAC:EquipmentConfiguration object.
+       \required-field
+       \type object-list
+       \object-list ZoneNames
+   A3, \field Zone Equipment Object Type
+       \required-field
+       \type choice
+       \key ZoneHVAC:TerminalUnit:VariableRefrigerantFlow
+       \key ZoneHVAC:AirDistributionUnit
+       \key ZoneHVAC:EnergyRecoveryVentilator
+       \key ZoneHVAC:EvaporativeCoolerUnit
+       \key ZoneHVAC:HybridUnitaryHVAC
+       \key ZoneHVAC:ForcedAir:UserDefined
+       \key ZoneHVAC:FourPipeFanCoil
+       \key ZoneHVAC:OutdoorAirUnit
+       \key ZoneHVAC:PackagedTerminalAirConditioner
+       \key ZoneHVAC:PackagedTerminalHeatPump
+       \key ZoneHVAC:UnitHeater
+       \key ZoneHVAC:UnitVentilator
+       \key ZoneHVAC:VentilatedSlab
+       \key ZoneHVAC:WaterToAirHeatPump
+       \key ZoneHVAC:WindowAirConditioner
+       \key ZoneHVAC:Baseboard:RadiantConvective:Electric
+       \key ZoneHVAC:Baseboard:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:RadiantConvective:Steam
+       \key ZoneHVAC:CoolingPanel:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:Convective:Electric
+       \key ZoneHVAC:Baseboard:Convective:Water
+       \key ZoneHVAC:HighTemperatureRadiant
+       \key ZoneHVAC:LowTemperatureRadiant:VariableFlow
+       \key ZoneHVAC:LowTemperatureRadiant:ConstantFlow
+       \key ZoneHVAC:LowTemperatureRadiant:Electric
+       \key ZoneHVAC:Dehumidifier:DX
+       \key ZoneHVAC:IdealLoadsAirSystem
+       \key ZoneHVAC:RefrigerationChillerSet
+       \key Fan:ZoneExhaust
+       \key WaterHeater:HeatPump:PumpedCondenser
+       \key WaterHeater:HeatPump:WrappedCondenser
+       \key HeatExchanger:AirToAir:FlatPlate
+       \key AirLoopHVAC:UnitarySystem
+   A4, \field Zone Equipment Name
+       \required-field
+       \type object-list
+       \object-list ZoneEquipmentNames
+   A5, \field Zone Equipment Outlet Node Name
+       \note Only used for airflow equipment
+       \note The outlet node from the zone equipment that will be split to the spaces.
+       \type node
+   A6, \field Thermostat Control Method
+       \note SingleSpace satisfies the thermostat in the Control Space Name
+       \note Maximum satisfies the thermostat in the connected spaces with the highest deviation from setpoint
+       \note Ideal ignores the Space Output Fractions and distributes the output to match the current space sensible loads.
+       \type choice
+       \key SingleSpace
+       \key Maximum
+       \key Ideal
+       \default SingleSpace
+   A7, \field Control Space Name
+       \note Used with SingleSpace thermostat control method
+       \type object-list
+       \object-list SpaceNames
+   A8, \field 
```

Comment:
field Space Airflow Fraction Method

Reply:
> field Space Airflow Fraction Method

You don't like "Space Sizing Basis" which I deleted by mistake?  I see, Method is consistent with other objects such as DesignSpecification:ZoneHVAC:Sizing "Cooling Supply Air Flow Rate Method".
How about:

Space Fraction Method
Space 1 Name
Space 1 Fraction
Space 1 Supply Node Name
etc.

Reply:
I do like Method better, but Space Sizing Basis isn't all that bad either. I think we use Method in many places to identify a choice of different ways to interpret things. So it's up to you for this one. Also, this relates only to air flow which is why I threw that in the name.

Reply:
It's intended to work for both airflow and non-airflow, like a baseboard heater (but there are kinks to work out there as well).

Reply:
OK, yeah, I was looking at the figure above and it does apply to more than air flow. I was also looking at the ideal choice to meet the space load. I like that one for sizing but I think your right that this will likely be a fraction of air flow or capacity to each space. That's more like a VAV TU in a zone would distribute to the spaces within that zone (i.e., my office, 1 of 4, that are connected to the TU upstream of all of us and we each have a diffuser.